velox/docs/functions/presto/aggregate.rst

```diff
@@ -82,12 +82,12 @@ General Aggregate Functions
     each input value occurs. Supports integral, floating-point,
     boolean, timestamp, and date input types.
 
-.. function:: geometric_mean(x) -> double
+.. function:: geometric_mean(bigint) -> double
```

Comment:
I don't think this is a valid syntax. Would you generate the docs and confirm the formatting?

Reply:
The result is:
![image](https://github.com/facebookincubator/velox/assets/379733/f7418d3d-f1f0-4192-bfde-b03763ada7ff)

@mbasmanova does this seem right to you?

Reply:
Looks good. Thanks.